UnityProject/Assets/Scripts/Strings/ChatTemplates.cs

```diff
@@ -5,12 +5,12 @@ namespace Strings
 {
 	public static class ChatTemplates
 	{
-		public const int SmallText = -2;
-		public const int NormalText = 1;
-		public const int LargeText = 8;
-		public const int VeryLargeText = 16;
-		public const int ExtremelyLargeText = 26;
-		public const int AnnoyingText = 48;
+		public const int SmallText = 12;
```

Comment:
if I understand this correctly, size used to be ``-2`` but now it is ``12 + FONTSCALE``. So it still won't be the same it used to be. Are you sure you don't want to multiply instead?

Reply:
ignore this change